ptbcontrib/chat_to_link/chat_to_link.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+#
+# A library containing community-based extension for the python-telegram-bot library
+# Copyright (C) 2020-2021
+# The ptbcontrib developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains a helper function to get joinable links from chats."""
+from typing import Optional
+from telegram import Chat
+from telegram.error import BadRequest
+
+
+def get_chat_link(chat: Chat) -> Optional[str]:
```

Comment:
just one last question: Is there a reason why you named the modle `chat_to_link` but the function `get_chat_link`?

Reply:
not really. I'm open to any suggestions though.

Reply:
I'd just prefer to keep the naming consistent, but don't have a favorite among the two …